### PR TITLE
Changed more Shotgun mag sizes

### PR DESCRIPTION
**Pull request by GuiShing**
Credit to Efdee for the suggestions.
Benelli Nova should have 7 + 1
M4 Super 90 should have 7 + 1
Remington 870 should have 6 + 1
M1 Super 90 should have 6 + 1
Badguy M4 Super90 should have 5+1 since it's based on the civilian version.
https://en.wikipedia.org/wiki/Benelli_M4

### CODE REVIEW

File: System/SwatEquipment.ini [hunk 1]
```diff
@@ -3918,7 +3918,7 @@ PlayerUsable=true
 EnemyUsesAmmo=(AmmoClass="SwatAmmo.M870SGAmmo",Chance=60)
 EnemyUsesAmmo=(AmmoClass="SwatAmmo.M870SG000Ammo",Chance=30) 
 EnemyUsesAmmo=(AmmoClass="SwatAmmo.M870SGSabotAmmo",Chance=10)
-MagazineSize=5
```
Comment (T-MaxWiese-T) [inline System/SwatEquipment.ini:3921]: There are two options. This value is also realistic

File: System/SwatEquipment.ini [hunk 2]
```diff
@@ -4089,7 +4089,7 @@ PlayerUsable=true
 EnemyUsesAmmo=(AmmoClass="SwatAmmo.NovaPumpSGAmmo",Chance=60)
 EnemyUsesAmmo=(AmmoClass="SwatAmmo.NovaPumpSG000Ammo",Chance=30)
 EnemyUsesAmmo=(AmmoClass="SwatAmmo.NovaPumpSGSabotAmmo",Chance=10)
-MagazineSize=7
```
Comment (T-MaxWiese-T) [inline System/SwatEquipment.ini:4092]: The website says 4+1

File: System/SwatEquipment.ini [hunk 3]
```diff
@@ -4165,7 +4165,7 @@ PlayerUsable=true
 EnemyUsesAmmo=(AmmoClass="SwatAmmo.M870SGAmmo",Chance=60)
 EnemyUsesAmmo=(AmmoClass="SwatAmmo.M870SG000Ammo",Chance=30)
 EnemyUsesAmmo=(AmmoClass="SwatAmmo.M870SGSabotAmmo",Chance=10)
-MagazineSize=5
```
Comment (T-MaxWiese-T) [inline System/SwatEquipment.ini:4168]: This value is also realistic

File: System/SwatEquipment.ini [hunk 4]
```diff
@@ -4247,7 +4247,7 @@ OriginalVariant=class'SwatEquipment.Remington870SG'
...
```
Comment (T-MaxWiese-T) [inline System/SwatEquipment.ini:4250]: This value is also realistic

File: System/SwatEquipment.ini [hunk 5]
```diff
@@ -6170,7 +6170,7 @@ Dc2=0.001949
 
 [SwatEquipment.BadguyM4Super90SG]
 ;RoundBasedWeapon configuration
-MagazineSize=5
```
Comment (T-MaxWiese-T) [inline System/SwatEquipment.ini:6173]: Right

File: System/SwatEquipment.ini [hunk 6]
```diff
@@ -4566,7 +4566,7 @@ ZoomedAutoRecoilBase=190
 PlayerUsable=true
 
 ;RoundBasedWeapon configuration
-MagazineSize=7
+MagazineSize=8
```
Comment (T-MaxWiese-T) [inline System/SwatEquipment.ini]: Wrong

File: System/SwatEquipment.ini [hunk 7]
```diff
@@ -4665,7 +4665,7 @@ Range=4000
 ReloadAnimationRate=1.0
 UseAnimationRate=1.75
 Choke=2.39
-MagazineSize=5
+MagazineSize=7
```
Comment (T-MaxWiese-T) [inline System/SwatEquipment.ini]: Wrong

File: System/SwatEquipment.ini [hunk 8]
```diff
@@ -3995,7 +3995,7 @@ PlayerUsable=true
 EnemyUsesAmmo=(AmmoClass="SwatAmmo.M4Super90SGAmmo",Chance=60)
 EnemyUsesAmmo=(AmmoClass="SwatAmmo.M4Super90SG000Ammo",Chance=30)
 EnemyUsesAmmo=(AmmoClass="SwatAmmo.M4Super90SGSabotAmmo",Chance=10)
-MagazineSize=7
+MagazineSize=8
```
Comment (T-MaxWiese-T) [inline System/SwatEquipment.ini]: Right

**Review by T-MaxWiese-T — CHANGES_REQUESTED** [review]

File: System/SwatEquipment.ini (hunk 4 above)
Comment (GuiShing) [reply System/SwatEquipment.ini:4250]: Law enforcement would use 6+1, though

File: System/SwatEquipment.ini (hunk 2 above)
Comment (GuiShing) [reply System/SwatEquipment.ini:4092]: The nova has an extended tube with 7+1, but swat 4 doesn't have it modeled.

File: System/SwatEquipment.ini (hunk 1 above)
Comment (GuiShing) [reply System/SwatEquipment.ini:3921]: Law enforcement would  prefer 6+1 when it comes to combat shotguns

**Pull request closed without merging**